src/components/_modal/pages/material/Material.page.js

```diff
@@ -378,13 +379,15 @@ const Material = ({ context }) => {
         href={getWorkUrl(title, creators, workId)}
       >
         <Text className={styles.link} type="text2" tag="span">
-          Gå til bogen
+          <Translate context="profile" label="goToBook" />
```

Comment:
det viste jeg ikke vi kunne! 💡

Reply:
..først vidste jeg ikke hvad du mente, men kan se det er første gang i projektet det er gjort. (troede jeg selv havde gjort sådan her for det meste - men nej). Ja man kan lave et direkte komponent